lib/components/Input/text-field/text-field.stories.tsx

```diff
@@ -0,0 +1,160 @@
+                type: "radio",
+            },
+            options: ["neutral", "success", "error"],
+        },
+        variant: {
+            control: {
+                type: "radio",
+            },
+            options: ["fill", "outline"],
+        },
+        label: {
+            control: {
+                type: "text",
+            },
+        },
+        textAlignment: {
+            control: {
+                type: "radio",
+            },
+            options: ["left", "center"],
+        },
+        leftStatusMessage: {
+            control: {
+                type: "text",
+            },
+        },
+        rightStatusMessage: {
+            control: {
+                type: "text",
+            },
+        },
+    },
+} satisfies Meta<typeof TextField>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const DefaultTextField: Story = {
+    args: {
+        placeholder: "Placeholder",
+    },
+};
+export const SuccessTextField: Story = {
+    args: {
+        placeholder: "Placeholder",
+        variant: "outline",
+        status: "success",
+    },
+};
+export const ErrorTextField: Story = {
+    args: {
+        placeholder: "Placeholder",
+        variant: "outline",
+        status: "error",
+    },
+};
+export const DisabledTextField: Story = {
+    args: {
+        placeholder: "Placeholder",
+        disabled: true,
+        variant: "outline",
+    },
+};
+
+export const TextFieldWithLabel: Story = {
+    args: {
+        placeholder: "Placeholder",
+        label: "Label",
+    },
+};
+
+export const SuccessStatusIconTextField: Story = {
+    args: {
+        placeholder: "Placeholder",
+        variant: "outline",
+        status: "success",
+        statusIcon: StandaloneCircleCheckBoldIcon,
+    },
+    argTypes: {
+        status: {
+            table: {
+                disable: true,
+            },
+        },
+    },
+};
+
+export const StatusMessageTextField: Story = {
+    args: {
+        placeholder: "Placeholder",
+        variant: "outline",
+        status: "success",
+        leftStatusMessage: "Status message goes here",
+    },
+};
+
+export const DoubleStatusMessageTextField: Story = {
+    args: {
+        placeholder: "Placeholder",
+        variant: "outline",
+        status: "success",
+        leftStatusMessage: "Status message goes here",
+        rightStatusMessage: "0/0",
+    },
+};
+
+export const SuccessMessageTextFieldWithIcons: Story = {
+    args: {
+        placeholder: "Placeholder",
+        variant: "outline",
+        status: "success",
+        leftStatusMessage: "Status message goes here",
+        rightStatusMessage: "0/0",
+        statusIcon: StandaloneCircleCheckBoldIcon,
+    },
+};
+export const ErrorMessageTextFieldWithIcons: Story = {
+    args: {
+        placeholder: "Placeholder",
+        variant: "outline",
+        status: "error",
+        leftStatusMessage: "Status message goes here",
+        rightStatusMessage: "0/0",
```

Comment:
a suggestion for future maintainability: we can create variables for all the repeated strings in the upper scope and pass them to the stories like this :) :
```suggestion
        placeholder, variant, status, leftStatusMessage, rightStatusMessage,
```

Reply:
created constant for statuses and variants